pkg/authorization/authz.go

```diff
@@ -77,10 +89,16 @@ func (ctx *Ctx) AuthZRequest(w http.ResponseWriter, r *http.Request) error {
 	}
 
 	for _, plugin := range ctx.plugins {
+		if isValInList(plugin.Name(), blacklist) {
+			continue
+		}
 		logrus.Debugf("AuthZ request using plugin %s", plugin.Name())
 
 		authRes, err := plugin.AuthZRequest(ctx.authReq)
 		if err != nil {
+			if err == ErrInvalidPlugin {
+				blacklist = append(blacklist, plugin.Name())
```

Comment:
what's the workflow to remove a plugin from the blacklist?  I'm worried that this might be an unexpected and silent failure for users